galaxy_importer/collection.py

```diff
@@ -173,12 +232,105 @@ def _load_collection_manifest(self):
         if not os.path.exists(manifest_file):
             raise exc.ManifestNotFound('No manifest found in collection')
 
+        default_logger.debug('manifest_file: %s', manifest_file)
+
         with open(manifest_file, 'r') as f:
             try:
                 data = schema.CollectionArtifactManifest.parse(f.read())
             except ValueError as e:
                 raise exc.ManifestValidationError(str(e))
-            self.metadata = data.collection_info
+
+            default_logger.debug('data: %s', data)
+            default_logger.debug('data.file_manifest_file: %s', data.file_manifest_file)
+            return data
+
+    def _load_collection_file_manifest(self, path_prefix, file_manifest_file):
```

Comment:
I don't think we should introduce names `file_manifest` `FileManifest` when `manifest_file` is already in use, and people equate "manifest" to `MANIFEST.json` not `FILES.json`. Can these be changed to a synonym like `file[s]_listing` `file[s]_record` `file[s]_register` `files_json`?

If a var/class does not refer somehow to `MANIFEST.json` then I don't think it should be named "manifest"